src/tools/genpolicy/tests/testdata/state/execprocess/pod.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: busybox
+spec:
+  runtimeClassName: kata-cc
+  containers:
+    - name: first-test-container
```

Comment:
Another container with a different command would make sense, and then test whether container A' command is rejected for container B.

Reply:
Thanks I will add this test case.

Reply:
- Added `second_test_container` in pod.yaml with exec args
- added test case to validate first_test_container's exec args request for second_test_container